pyqg/xarray_output.py

```diff
@@ -126,13 +104,30 @@ def model_to_dataset(m):
     # Create a dictionary of coordinates
     coordinates = {}
     for cname in coord_database:
-        if hasattr(m, cname):
-            if cname in transformations:
-                data = transformations[cname](m)
-            else:
-                data = getattr(m, cname)
-            coordinates[cname] = (coord_database[cname], data, coord_attr_database[cname])
+        data = transformations[cname](m).copy()
+        coordinates[cname] = (coord_database[cname], data, coord_attr_database[cname])
+
+    # Create a dictionary of global attributes
+    global_attrs = {}
+    for aname in attribute_database:
+        if hasattr(m, aname):
+            data = getattr(m, aname)
+            long_name = attribute_database[aname]
+            global_attrs[long_name] = (data)
```

Comment:
Hmmm, I'm wondering about this. The problem with using `long_name` as the key here is that you lose the the actual name of the parameter. That would make it very hard to reinitialize a model based on the pickup file.

A different option would be to just use `aname` as the key and discard `long_name`.

Reply:
Thanks for the feedback and that's a good point. I've reverted back to using the parameter key name and made `attribute_database` a list instead of a dictionary. I also changed the test to reflect this and now it passes.